content/en/serverless/aws_lambda/configuration.md

```diff
@@ -501,87 +501,11 @@ If you are using a runtime or custom logger that isn't supported, follow these s
 
 ## Link errors to your source code
 
-<div class="alert alert-info">This feature supports Go, Java, Python, and JavaScript.</div>
+[Datadog source code integration][26] allows you to link your telemetry (such as stack traces) to the source code of your Lambda functions in GitHub. 
```

Comment:
Suggesting this since we don't only support GitHub
```suggestion
[Datadog source code integration][26] allows you to link your telemetry (such as stack traces) to the source code of your Lambda functions in your Git repositories. 
```